joomla/joomla-cms
# [4.0] Re-add the colour state for publish dropdown

Pull Request for Issue # .

### Summary of Changes

In Joomla 3, when selecting the state of something via the dropdown (published, unpulishged, trashed), the colour of the dropdown changed depending on the option selected (red/green).

This re-add the JS, providing the same functionality

### Testing Instructions

- Go to Module Manager. Open an existing module or create a new one.
- On the right hand side, switch the **State** and notice the colour change.
